docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
```

Comment:
Adding a docker-compose and in general improving the devx for contributors could be done in a separate issue/pr, so I would remove this file from the PR!